packages/extension/src/background/conditional-new-tab.test.js

```diff
@@ -0,0 +1,11 @@
+import { Extension } from '@mujo/utils'
+import { NEW_TAB } from '../constants'
+import { onConditionalNewTab } from './conditional-new-tab'
+
+const { storage } = Extension
+
+test('conditional new tab', async () => {
+  onConditionalNewTab(NEW_TAB)
+
+  expect(storage.sync.get).toBeCalled()
```

Comment:
I would like to test that tab update is being called if the setting is true and not called if the setting is off.